### PR TITLE
Add openssh-client & bump versions

### PR DESCRIPTION
**Pull request by cyrille-leclerc**
Add openssh-client,
Bump: selenium -> 2.49.1, azure-cli:0.9.14, kubectl -> 1.1.4, oc -> 1.1.1

### CODE REVIEW

**Comment by cyrille-leclerc** [comment]
cc @ydubreuil @ndeloof @michaelneale

File: README.md [hunk 1]
```diff
@@ -18,63 +18,52 @@ We have decided to bundle many tools in the same imge to cover as many Java use
...
+-   gcc (latest): 4.9.2-10ubuntu13
+-   Java: OpenJDK 8 (latest): 1.8.0_45
+-   Maven located in `/usr/share/maven/`: 3.3.9
+-   Node.js: 4.2.4
+-   Bower: 1.7.2
+-   Grunt CLI: 0.1.13
+-   Gulp: 3.9.0
+-   Web Browser tests: XVFB (latest)
+-   Firefox (latest) at `/usr/bin/firefox`: Mozilla Firefox 43.0.4
+-   Selenium at `/opt/selenium/selenium-server-standalone.jar`: 2.48.2
```
Comment (ydubreuil) [inline README.md]: 2.49.1 instead?

File: README.md [hunk 2]
```diff
@@ -18,63 +18,52 @@ We have decided to bundle many tools in the same imge to cover as many Java use
...
+-   AWS CLI: 1.9.17
+-   Azure CLI: 0.9.14
+-   Cloud Foundry CLI (latest) at `/usr/local/bin/cf`
+-   Open Shift V3 CLI at `/usr/local/bin/oc`: 1.1.1.1
```
Comment (ydubreuil) [inline README.md]: 1.1.1 instead?

**Comment by ydubreuil** [comment]
Apart from minor fixes on changelog, :bee:

**Comment by cyrille-leclerc** [comment]
Thanks @ydubreuil I updated the readme and pushed to origin (instead of pushing to my PR)

**Pull request merged**